source/Engine.cpp

```diff
@@ -1488,7 +1488,12 @@ void Engine::CalculateStep()
 	Point newCenterVelocity;
 	if(flagship)
 	{
-		newCenter = flagship->Position();
+		if(flagship->IsThrusting())
+			offsetMultiplier = offsetMultiplier >= 1 ? 1.0 : offsetMultiplier + 0.0025;
```

Comment:
1.0 is the same as 1. (and is a double btw)

Reply:
done